### DISCUSSION TITLE
Disabling development mode in JetBrains IDE plugin

### DISCUSSION THREAD

**Discussion by orange-buffalo** [community/community#39116] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When using JetBrains IDE plugin (as described [here](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide)), Copilot starts in development mode. This occupies a popular port `8180` and blocks other applications that use this port from functioning properly:

![image](https://user-images.githubusercontent.com/4532247/201794932-6a1f9389-ebc4-4bac-9b58-29219c7dd27f.png)

There is no option to disable development mode nor to change the port. It prevents from using Copilot in many environments.

**Comment by cmp-nct** [comment]
Just curious, which application do you use that acts as a daemon on port 8180 ?

**Comment by orange-buffalo** [comment]
It's one of the company's product microservices.

**Comment by iamcjwilliams** [comment]
@orange-buffalo Thank you! Our copilot team is investigating this issue
